\documentclass[11pt, letterpaper]{article}
\usepackage[top=1in, bottom=1in, left=1in, right=1in]{geometry}

% Math, graphics, and bibliography
\usepackage{amsmath,amssymb,amsfonts,mathrsfs,mathtools}
\usepackage{graphicx}
\usepackage[round,numbers,sort&compress]{natbib}
\renewcommand{\bibnumfmt}[1]{#1.}

% My preferred fonts
\usepackage{lmodern}
\usepackage[T1]{fontenc}
\usepackage[sc]{mathpazo}
\usepackage{textcomp}

% For hyperref links
\usepackage[linktoc=all]{hyperref}
\hypersetup{
    colorlinks,
    citecolor=black,
    filecolor=black,
    linkcolor=black,
    urlcolor=black
}

% For fancier fractions and figure captions
\usepackage[labelfont={bf}, margin=1cm]{caption}
\usepackage{units}
\usepackage{booktabs}

% Make subsections more clear
\usepackage{titlesec}

\titleformat{\subsubsection}
  {\normalfont\itshape}{\thesubsubsection}{1em}{}

% For supplemental figures
\newcommand{\beginsupplement}{%
        \clearpage
        \setcounter{table}{0}
        \renewcommand{\thetable}{S\arabic{table}}%
        \setcounter{figure}{0}
        \renewcommand{\thefigure}{S\arabic{figure}}%
     }

% Puts the affiliations on the cover page
\usepackage{authblk}

\begin{document}
\title{Quantifying stochastic noise in cultured\\ circadian reporter cells}
\author[1]{Peter C. St. John}
\author[1,*]{Francis J. Doyle III}
\affil[1]{Department of Chemical Engineering, University of California Santa
Barbara, Santa Barbara, California 93106-5080}
\affil[*]{Email: \texttt{doyle@engineering.ucsb.edu}}
\date{\today}
\maketitle

\begin{center}
Running head:\\ {Quantifying circadian stochastic noise} \\[1ex]
Keywords:\\ Systems Biology | Circadian Rhythms \\ Gene
Regulatory Network | Stochastic | Synchronization
\end{center}

\pagebreak
\begin{abstract}
Stochastic noise at the cellular level has been shown to play a fundamental role in circadian oscillations, influencing how groups of cells entrain to external cues and likely serving as the mechanism by which cell-autonomous rhythms are generated.
Despite this importance, few studies have investigated how clock perturbations affect stochastic noise -- even as increasing numbers of high-throughput screens categorize how gene knockdowns or small molecules can change clock period and amplitude.
This absence is likely due to the difficulty associated with measuring cell-autonomous stochastic noise directly, which currently requires the careful collection and processing of single-cell data.
In this study, we show that the damping rate of population-level bioluminescence recordings can serve as an accurate measure of overall stochastic noise, and one that can be applied to future and existing high-throughput circadian screens.
Using cell-autonomous fibroblast data, we first show directly that higher noise at the single-cell results in faster damping at the population level.
Next, we show that the damping rate of cultured cells can be changed in a dose-dependent fashion by small molecule modulators, and confirm that such a change can be explained by single-cell noise using a mathematical model.
We further demonstrate the insights that can be gained by applying our method to a genome-wide siRNA screen, revealing that stochastic noise is altered independently from period, amplitude, and phase.
Finally, we hypothesize that the unperturbed clock is highly optimized for robust rhythms, as very few gene perturbations are capable of simultaneously increasing amplitude and lowering stochastic noise.
Ultimately, this study demonstrates the importance of considering the effect of circadian perturbations on stochastic noise, particularly with regard to the development of small-molecule circadian therapeutics.
\end{abstract}

\section*{Author Summary}

As most organisms exist in an environment that changes predictably with a 24-hour period, highly optimized genetic circuits turn on and off the production of key regulatory proteins to anticipate the day/night cycle.
In humans, the demands of a modern society have required that we deviate from this evolutionarily prescribed sleep and feeding schedule, resulting in increased long-term risks of metabolic disease.
There is therefore a desire to find pharmacological treatments that would restore the normal functioning of our circadian clock despite irregular behavioral schedules.
One aspect of these treatments that is often overlooked in searching for candidate drugs is how these treatments might affect the accuracy of the circadian timing system.
% Since each cell in the body keeps its own internal time, increasing the sloppiness of the clock machinery can result in tissues not being able to agree on what time to turn on protein production.
Recording the time of each cell is possible but difficult; as a result single-cell approaches cannot be scaled up to high-throughput searches.
In this paper, we show that it is possible to estimate how much the noise of a system has changed by looking only at the averaged protein production of an entire population of cells.
Such an approach allows us to analyze prior data from high-throughput screens, and show that the natural clock has been highly optimized to be both accurate and high amplitude.



\section*{Introduction}

Circadian rhythms are daily changes in gene expression and physiology that persist even in the absence of external environmental cues \cite{Herzog2007}.
In mammals, such rhythms are organized in a hierarchical fashion: at the tissue-level, the brain's suprachiasmatic nucleus (SCN) serves as the master pacemaker and keeps circadian oscillations in peripheral tissues in phase with the light-dark cycle.
In the SCN, cell-to-cell coupling keeps individual cells in tight synchrony \cite{Herzog2004}, while coupling between circadian oscillations in peripheral tissues {\itshape in vivo} or cultured reporter cells {\itshape in vitro} is thought to be very weak or absent entirely \cite{Guenthner2014, Noguchi2013}.
Within each tissue, cellular-level rhythms in gene transcription are generated by a large network of interacting gene regulatory elements, in which time-delayed transcription-translation negative feedback gives rise to sustained oscillations \cite{Ueda2005}.
The robust oscillation of circadian factors has been linked to metabolic health \cite{Bass2012}, since rhythms compromised by gene knockout \cite{Marcheva2010} or irregular feeding schedules \cite{Hatori2012} result in an increased risk of metabolic disease.
Additionally, as the amplitude of circadian transcription can be affected by lifestyle variables such as diet, age, or work schedule, there has been recent interest in developing pharmacological strategies for increasing the amplitude of circadian cycles in metabolic tissues \cite{Chen2013}.

A detailed understanding of the underlying transcriptional mechanisms is essential for the development of circadian therapeutics to be successful. 
The functional roles of different genes in circadian regulation have traditionally been studied using behavioral-level data and genetic knockout experiments \cite{Vitaterna1994}.
Bioluminescence-based cellular circadian reporters offer a more direct view of the gene regulatory network \cite{Balsalobre1998} and are amenable to high-throughput screens, allowing genome-wide exploration into factors that affect circadian rhythmicity \cite{Zhang2009}.
Additionally, cultured circadian reporter cells allow the change in transcriptional amplitude following a perturbation to be quantified.
This additional parameter has proven useful in differentiating between perturbations with the same effect on period \cite{St.John2014} and has aided the search for small-molecule therapeutics to boost clock amplitude \cite{Chen2013}.

Bioluminescence rhythms at the cell culture or tissue-level are the result of the collective behavior of thousands of cells.
Transcription at the single-cell level is strongly affected by {\itshape intrinsic} cellular noise, caused by the low molecular counts of the mRNA and protein species involved. 
As a result, bioluminescence traces of individual cells are stochastic, with significant variability in both amplitude and period length from cycle to cycle \cite{Welsh2004}. 
In addition to intrinsic noise, circadian oscillations are also affected by {\itshape extrinsic} noise sources.
Extrinsic noise results from heterogeneity between cells, such as differences in size or physical environment, leading to differences on a cell-to-cell basis in expected period and amplitude.
For circadian systems, intrinsic noise has been shown to play a larger role: a single cell's variability in period from cycle-to-cycle is larger than the variability in mean period length between cells \cite{Herzog2004}.
Both sources of noise have an effect on population-level rhythms: in cell cultures that lack cell-to-cell coupling, it has been shown that  stochastic noise is manifested in damped oscillations at the population-level as individual oscillators gradually drift out of phase \cite{Nagoshi2004, Welsh2004}.
This type of behavior has also been seen in other experimental systems, such as NF-$\kappa$B signaling or yeast glycolytic oscillations \cite{Nelson2004, Aon1992}.
The amount of noise in system is therefore linked to the ability of tissue-level clocks to maintain high-amplitude rhythms.

Despite the averaging that occurs at the population-level, cell-autonomous stochastic noise plays an important role in determining the function of the circadian oscillator.
Noise in circadian rhythms has long been considered an important factor in how circadian rhythms have evolved \cite{Barkai2000}.
A recent study showed that stochasticity is critical to the population-level response to a neuropeptide and forms the basis for how the SCN entrains to light-mediated cues \cite{An2013}.
Additional studies have suggested that the basis of single-cell rhythmicity may depend on stochastic noise, as models of deterministically damped oscillators, when simulated stochastically, capture the noise characteristics seen in single-cell fibroblast data equally well as limit-cycle oscillators \cite{Westermark2009}.
Despite the importance of single-cell stochasticity in circadian rhythms, measuring stochastic noise currently requires careful preparation, recording, and image processing of individual cells \cite{Leise2012}. 
As a result, while circadian perturbations have been postulated to affect single-cell stochasticity \cite{Rougemont2007}, no study has experimentally quantified changes to stochastic noise as a result of a small molecule or genetic perturbation.

In this study, we demonstrate that changes to stochastic noise can be reliably inferred from the changes to the damping rate of population-level bioluminescence recordings of cultured circadian reporters.
Our method assumes that oscillations in individual cells are both
\begin{itemize}
  \item independent (no significant cell-to-cell coupling) \cite{Noguchi2013, Guenthner2014},
  \item and sustained (do not damp on a single-cell basis) \cite{Welsh2004, Nagoshi2004},
\end{itemize}
which have been shown to hold experimentally for cultured fibroblast cells.
We demonstrate the validity and usefulness of such an approach on several types of circadian data.
First, we show using single-cell fibroblast data that intrinsic stochastic noise is directly related to population-level damping.
Next, we show that a small-molecule modulator is able to change damping rate in a dose-dependent fashion, and verify using a mathematical model that changes to intrinsic stochastic noise is a likely mechanism.
Finally, we calculate the genome-wide effects of siRNA knockdown on overall stochastic noise, and demonstrate that population-level damping rate is independent of other circadian parameters, such as period or amplitude.
Using this additional information, we show that circadian rhythms have likely evolved to an optimal point of high amplitude and low stochastic noise.
Our results should prove especially important in the future search for small molecule circadian therapeutics, as it allows the effect of candidate drugs on stochastic noise to be quantified in a high-throughput manner.


\section*{Results and Discussion}

\subsection*{Higher noise results in faster damping in population-level rhythms}

While both intrinsic and extrinsic noise sources can contribute to population-level damping, intrinsic noise is thought to play a more significant role in circadian systems \cite{Herzog2004}.
We therefore first sought to determine whether changes to intrinsic stochastic noise alone are sufficient to explain population-level changes in damping rate.
To do this, we calculated noise characteristics from experimental data on individual PER2::LUC fibroblast cells \cite{Leise2012}.
Cells were sorted into two groups, a low-noise group and a high-noise group, based on the relative high-frequency noise, period variability, and amplitude variability present in each trace.
Example rhythms from cells in both groups are shown in Figure~\ref{fig:fibroblast_noise}A.
Because the cells were not synchronized at the start of the recording, this effect is replicated {\itshape in silico} by shifting each series in time to align their start phases.
Population-level bioluminescence traces were then found by averaging the cellular PER2::LUC signal in each group.
Both populations displayed averaged rhythms which resembled a damped sinusoid, similar to those seen in bioluminescence recordings of entire cell cultures.
Fitting the averaged expression of each group with a damped sinusoid revealed that the low-noise group also had a lower damping rate (Figure~\ref{fig:fibroblast_noise}B).
The significance of this difference was confirmed via a bootstrap analysis (Figure~\ref{fig:fibroblast_noise}C), where cells were randomly assigned in each bootstrap trial to either the low-noise or the high-noise group.

\begin{figure}[tbp]
  \begin{center}
    % \includegraphics[width=0.8\textwidth]{figures/pdfs/Fig1.pdf}
  \end{center}
  \caption{{\bfseries Single-cell bioluminescence recordings show that higher stochastic noise results in faster damping at the population-level.}
  Data on the bioluminescence of single-cell fibroblasts was taken from Leise {\itshape et al.,} 2012 \cite{Leise2012}.
({\bfseries A}) Cells were sorted into two groups depending on their degree of stochastic noise. An example trace from each of the two groups is shown, demonstrating different levels of noise present in the dataset.
({\bfseries B}) After artificially synchronizing each cell, we calculate averaged bioluminescence rhythms of each group (solid lines). A damped sinusoid fit to both groups reveals a difference in damping rate, demonstrated by fitted envelope functions ($\pm A\exp{-dt}$, dashed lines).
({\bfseries C}) The observed absolute difference in damping rate was shown to be significant ($p = 0.0264$) using $10,000$ bootstrap trials.}
\label{fig:fibroblast_noise}
\end{figure}


\subsection*{Clock perturbations can change single-cell stochastic noise}

\subsubsection*{A small molecule causes dose-dependent changes in the population-level damping rate of cultured cells}

We next demonstrate that perturbations to the transcriptional oscillator are capable of altering population-level damping rate.
The actions of small-molecule circadian modulators KL001 and longdaysin are well-characterized, and are known to affect circadian period and amplitude in a dose-dependent fashion \cite{St.John2014}.
By fitting experimental data on the population-level responses to increasing dosages of each molecule with a damped sinusoid, we show that KL001, but not longdaysin, increases damping rate in a dose-dependent fashion (Figure~\ref{fig:dose_dependence},~\ref{fig:small_molecule_r2}).
This change in damping rate is consistent across both reporter systems ({\itshape Bmal1-dLuc} and {\itshape Per2-dLuc} U2OS cells), indicating it is a fundamental property of the overall gene regulatory network.
This result indicates that stochastic noise can be altered by perturbations known to affect the transcription-translation feedback loop.

\begin{figure}[tbp]
  \begin{center}
    % \makebox[\textwidth][c]{\includegraphics[]{figures/pdfs/Fig2.pdf}}
  \end{center}
  \caption{{\bfseries Small molecule modulator KL001 increases damping rate in a dose-dependent fashion.}
  Experimental data on the dose-dependent effects of small molecules KL001 and longdaysin on cultured circadian reporter cells was taken from Hirota {\itshape et al.,} 2012 \cite{Hirota2012}.
({\bfseries A}) Detrended bioluminescence signals from the two reporter systems and two small molecules are shown normalized by the fitted amplitude, period, and phase. The normalized bioluminescence highlights the dose-dependent change in damping rate seen with the KL001 application (top), but not with longdaysin (bottom).
({\bfseries B}) Quantification of the dose-dependent change in damping rate caused by small molecule modulators. While both molecules lead to a dose-dependent increase in period, only KL001 shows a reliable change in damping rate.}
\label{fig:dose_dependence}
\end{figure}

\subsubsection*{Mathematical model predicts KL001 damping rate increase comes from increased single-cell stochastic noise}

We next test the hypothesis that the dose-dependent changes in damping rate from KL001 is due to changes in intrinsic (cell-autonomous) noise characteristics.
To do this, we employed a mathematical model of circadian rhythms previously used to explain the effects of both small molecule perturbations \cite{St.John2014}, summarized in Tables \ref{tab:model}-\ref{tab:parset}.
In order to capture changes to noise characteristics, we first converted the model to a stochastic biochemical system.
Population-level rhythms were generated by averaging the trajectories of $1,000$ individual, noninteracting oscillators.
The only free parameter in converting the existing deterministic model to a stochastic one is the cell volume, which was determined by fitting the observed population-level damping rate to that of the experimental control traces (Figure~\ref{fig:vol_calibration}).

The model was then used to predict the effects of KL001 and longdaysin administration on single-cell noise and population damping rates (Figure~\ref{fig:simulation}A).
Reductions in parameters previously attributed to the activities of each small molecule caused dose-dependent changes in period and damping rate at the population-level that closely matched experimental results (Figure~\ref{fig:simulation}B).
As the model includes no cell-to-cell communication or heterogeneity, this difference is manifested solely by changing the noise characteristics of individual cells.
This quantitative prediction by the mathematical model lends further support to the assumptions that individual cells are independent and sustained oscillators, and intrinsic noise plays the dominant role in determining population-level damping.

\begin{figure}[tbp]
  \begin{center}
    % \makebox[\textwidth][c]{\includegraphics[]{figures/pdfs/Fig3.pdf}}
  \end{center}
  \caption{{\bfseries Mathematical model accurately predicts dose-dependent changes in damping rate.}
({\bfseries A}) Example single-cell trajectories (top) and population-averaged trajectories (bottom, mean of $1,000$ cells) of cells under various treatments. Cells with the nominal parameter set (left, black) closely match the experimental damping rate for unperturbed cells. Cells with simulated KL001 action (red, center) are noisier at the single-cell level, and show faster damping at the population-level. Cells with simulated longdaysin action show slightly more accurate single-cell rhythms, with a corresponding reduction in the population-level damping rate.
({\bfseries B}) The model accurately predicts the general trend of period vs.\ damping rate for both KL001 and longdaysin perturbations. Experimental data points represent the mean of two replications at each concentration. Computational data points represent the mean of ten independent population simulations.}
\label{fig:simulation}
\end{figure}


\subsection*{Genome-wide effects of siRNA knockdown on single-cell stochastic noise}

Unlike from using single-cell imaging, inferring stochastic noise from the desynchronization rate of population-level recordings can be applied to existing and future high-throughput circadian screens.
We demonstrate the insights that can be gained from such an approach by analyzing the publicly available genome-wide siRNA screen from Zhang {\itshape et al.,} 2009 \cite{Zhang2009}.
The results of fitting a damped sinusoid to each of the $111,743$ bioluminescence trajectories is shown in Figure~\ref{fig:fit_quality}, in which $86\%$ of fits had an $R^2 > 0.8$.
Since sinusoidal parameters can only be confidently inferred for fits in which the $R^2$ was sufficiently high, wells were removed from further analysis if $R^2 < 0.8$.
Additionally, of the fits with a high $R^2$ value, only a small minority ($0.1\%$) had a negative damping rate.
This trend supports the assumption that intercellular synchronization is unlikely in cultured U2OS cells. 

\begin{figure}[tbp]
  \begin{center}
    % \includegraphics[]{figures/pdfs/Fig4.pdf}
  \end{center}
  \caption{{\bfseries Fit quality vs.\ damping rate for the genome-wide siRNA screen.}
({\bfseries A}) A plot of the $111,743$ individual fits shows that the majority of fits have a high $R^2$ value and positive damping rate. Only fits with $R^2 > 0.8$ were kept for further analysis.
({\bfseries B}) Examples chosen randomly from each of the four quadrant regions in (A). Sinusoidal parameters for fits 1-2 can be more confidently inferred than those for fits 3-4.}
\label{fig:fit_quality}
\end{figure}

We next checked how parameters varied on a plate-to-plate and well-to-well basis.
Well-to-well variation was relatively absent, aside from expected variation due to long- and short-period controls (Figure~\ref{fig:well_variation}).
Fits were normalized to remove plate-to-plate variation (Figure~\ref{fig:plate_variation}) using a robust z-score \cite{Birmingham2009}.
Additionally, we separated wells into a ``perturbed'' category and ``control'' category, depending on whether or not the well contained an siRNA perturbation.
As we show in Figure~\ref{fig:fit_distributions}, all fitted parameters displayed normal-like distributions, in which the control distributions showed much tighter clustering around the most likely values.
Quantifications of the mean, variance, skewness, and kurtosis for each distribution are shown in Table~\ref{tab:fit_distributions}.

\begin{figure}[tbp]
  \begin{center}
    % \includegraphics[width=0.8\textwidth]{figures/pdfs/Fig5.pdf}
  \end{center}
  \caption{{\bfseries Distributions in fitted parameters for the genome-wide siRNA screen.} For each well in the high-throughput screen, the period, amplitude and damping rate are calculated. After normalization, distributions in robust z-scores closely resemble normal distributions. For all parameters, the region of highest density is consistent between the control and perturbed populations, indicating many perturbations do not appreciably change clock dynamics.} 
\label{fig:fit_distributions}
\end{figure}

\begin{table}
  \begin{center}
    \begin{tabular}{lrrrrrr}\toprule
      {} & \multicolumn{2}{c}{$T$} & \multicolumn{2}{c}{$\ln A$} & \multicolumn{2}{c}{$d$} \\
      {}         & \multicolumn{1}{c}{C}         & \multicolumn{1}{c}{P}           & \multicolumn{1}{c}{C}         & \multicolumn{1}{c}{P}               & \multicolumn{1}{c}{C}         & \multicolumn{1}{c}{P}           \\\midrule
      Mean     & $-0.234$ & $0.187$ & $0.443$  & $-0.343$ & $0.043$  & $0.090$    \\
      Variance & $ 0.599$ & $3.313$ & $0.605$  & $ 3.074$ & $0.771$  & $2.850$    \\
      Skewness & $ 0.153$ & $0.367$ & $-1.823$ & $-0.580$ & $-0.107$ & $0.371$    \\
      Kurtosis & $ 3.772$ & $0.591$ & $8.329$  & $ 0.476$ & $2.423$  & $0.373$    \\
      \bottomrule
    \end{tabular}
  \end{center}
  \caption{{\bfseries Moments of the fitted parameter distributions (Figure~\ref{fig:fit_distributions}) after normalization and outlier removal.} Parameters were normalized by subtracting the median and dividing by the median absolute deviation on a plate-by-plate basis. Control distributions had less variance, were less skewed, and were more peaked then their perturbed counterparts.}
  \label{tab:fit_distributions}
\end{table}

\subsubsection*{Damping rate is independent of other sinusoidal parameters}

Since an siRNA's effect on period length does not effectively predict its effect on amplitude or phase (and vice-versa), we hypothesized that damping rate would similarly be independently affected by siRNA perturbations.
Low Pearson correlation coefficients between normalized parameter distributions defend this hypothesis, with the highest correlations among variables seen between amplitude and damping rate ($\rho = 0.285$, Table~\ref{tab:corr}).
Additionally, a multivariate linear regression of damping rate as a function of period, amplitude, phase, and perturbation type (control or perturbed, a categorical variable) produced an $R^2$ value of only $0.169$ (Table~\ref{tab:ols_reg}).
These results reinforce the notion that the population-level damping rate describes an independent feature of the underlying oscillator.

\begin{table}
  \begin{center}
    \begin{tabular}{rrrrr}
      \toprule
      {}       & $d$    & $\ln A$ & $T$    & $\theta$ \\\midrule
      $d$      & $1.000 $ & $0.285 $  & $-0.142$ & $-0.269$\\
      $\ln A$  & $0.285 $ & $1.000 $  & $-0.022$ & $-0.112$\\
      $T$      & $-0.142$ & $-0.022$  & $1.000 $ & $-0.113$\\
      $\theta$ & $-0.269$ & $-0.112$  & $-0.113$ & $1.000 $\\
      \bottomrule
    \end{tabular}
  \end{center}
  \caption{{\bfseries Correlation among normalized parameters of the high-throughput siRNA screen.} Pearson correlation coefficients are relatively low between fitted parameters, indicating that changes to damping rate (and thereby stochastic noise) are not explained by changes to period, amplitude, or phase.}
  \label{tab:corr}
\end{table}


\subsubsection*{The unperturbed clock lies at the Pareto frontier of high amplitude and low noise}

The ability of a population of oscillators to maintain robust oscillations is a function both of its initial amplitude as well as its damping rate.
A robust oscillator is therefore one with high initial amplitude and low stochastic noise.
Scatter plots of the control and perturbed distributions are shown in Figure~\ref{fig:outlier_dist}A-B, which indicate that few outlier points reside in the lower damping rate, higher amplitude quadrant.
In order to find perturbations which confidently change robustness, we grouped the siRNA perturbations by target gene and performed a two-sample Hotelling's $T^2$ test against the control population.
The resulting significant gene perturbations ($75\%$ of all genes) are shown in Figure~\ref{fig:outlier_dist}C.
Quantifying the distribution in outliers by quadrant, it is clear that only a small minority of perturbations ($3.3\%$) simultaneously increase amplitude and decrease stochastic noise.
It therefore appears that the unperturbed clock optimizes some trade-off between high amplitude and low stochastic noise, such that it is unlikely the knockdown of a single gene will make oscillations more robust. 

\begin{figure}[tbp]
  \begin{center}
    % \includegraphics[width=0.65\textwidth]{figures/pdfs/Fig6.pdf}
  \end{center}
  \caption{{\bfseries Effects of siRNA knockdowns on amplitude and damping rate.}
  Clock robustness is a function of both amplitude and damping rate.
  Distributions in amplitude and damping rate for control wells ({\bfseries A}) or perturbed wells ({\bfseries B}) indicate that perturbations tend to shift the clock towards regions of higher damping rate or lower amplitude. Green dots in each figure indicates the mean of the control population.
({\bfseries C}) Averaged effect of siRNA knockdown after grouping the perturbed population by Gene ID. Only those genes which were significantly different from the control distribution are shown (Hotelling's $T^2$ test, $\alpha = 0.01$).
({\bfseries D}) Radial histogram of the significant gene perturbations shown in (C). The area of each slice is proportional to the frequency of perturbations away from the mean in that direction. Very few gene knockdowns result in both higher amplitude and lower damping rate (red slices, lower right quadrant).
}
\label{fig:outlier_dist}
\end{figure}

\subsection*{Effect of cell heterogeneity}

In the preceding sections, we have demonstrated that changes to single-cell intrinsic noise are sufficient to explain the observed changes in population-level damping.
However, experimental work has shown that cell-autonomous fibroblast cells have a distribution of mean free-running periods \cite{Leise2012}.
Indeed, prior to the availability of single-cell data, studies explored the possibility that differences in mean periods served as the mechanism behind population-level damping \cite{Izumo2003}.
While it is true that the dephasing of a group of oscillators can be caused by both variance in the mean period as well as cycle-to-cycle variability, intrinsic stochastic noise likely plays a more significant role: we show that there is greater variance in period on a cycle-to-cycle basis than on a cell-to-cell basis in cultured fibroblast cells (Figure~\ref{fig:cell2cell-variability}).
This result has also been observed in dispersed SCN neurons, indicating cell heterogeneity is less severe than cell-autonomous noise \cite{Herzog2004}.

However, it is possible that damping rate changes due to siRNA or small molecule perturbation could be manifested through altering the system's extrinsic noise.
Such a change, caused perhaps by an unequal uptake of drug across cells, would result increased cell heterogeneity, and different dephasing kinetics.
While differentiating between intrinsic and extrinsic noise sources from population-level data is possible in theory (Figure~\ref{fig:dephasing-rates}), these differences are likely not identifiable from typical data (Figure~\ref{fig:computational_dephasing}).
Differences in damping rates are therefore best viewed as representative of changes to overall stochastic noise from both intrinsic and extrinsic factors.
Since both types of noise are important to determining the overall function of population-level rhythms, damping rates are still a valuable method of quantifying stochastic noise.


\section*{Materials and Methods}

\subsection*{Fitting a damped sinusoid to experimental data}
Raw experimental data $x(t_i), \  i \in \{0, \dots, N-1\}$ are first detrended using Hodrick-Prescott filter with a smoothing parameter $\gamma = 0.05\left(\nicefrac{24 \text{ hrs}}{T_s}\right)^4$, in which $T_s$ is the sampling rate (in hours) \cite{Ravn2002}.
The detrended data are then filtered using a low-pass filter to remove high-frequency noise (forward-backward Butterworth filter with $n = 5,\ w_c = 0.1$).
We denote the detrended and filtered experimental data by $y(t_i)$.
A damped sinusoid, specified by:
\[
  \hat{y}(t) = A e^{-d t} \sin\left(\frac{2\pi t}{T} + \theta\right)
\]
is then fit to the filtered data.
For numerical efficiency, the period, $T$, and damping rate, $d$, parameters are fit first using a matrix pencil method \cite{Hua1990}, reviewed in \cite{Zielinski2011}.
Amplitude, $A$, and phase, $\theta$, parameters are subsequently fit using a linear least-squares regression.
Note that in this manuscript we use amplitude to denote the initial rhythm strength, and damping rate to denote the rate at which this strength decays with time.
Overall $R^2$ values for the regression were calculated from the residual error between the detrended data and fitted sinusoid:
\[
  R^2 = 1 - \frac{\sum_{i = 0}^{N-1} (y(t_i) - \hat{y}(t_i))^2}{\sum_{i = 0}^{N-1} (y(t_i) - \bar{y}(t_i))^2}
\]
in which $\bar{y}(t_i)$ represents the mean of the detrended data.


\subsection*{Processing single-cell bioluminescence data}

Single-cell bioluminescence data for 79 cells was obtained from Leise {\itshape et al.,} 2012 \cite{Leise2012}.
As was done in the original study, a discrete wavelet transform (using PyWavelets, \url{http://www.pybytes.com/pywavelets}) was performed to detrend and remove noise.
A discrete wavelet transform decomposes the signal into multiple frequency bands \cite{Leise2011}. 
By only considering frequency bands close to the circadian frequency, high-frequency noise and low-frequency baseline oscillations can be removed.

\subsubsection*{Sorting cells by noise level}
As in the original study, various parameters describing the average noise level of each cell were collected.
Traces were denoised and detrended by keeping only the $(8 \text{hr}, 258 \text{hr})$ wavelet components -- resulting in rhythms which only contained oscillations with periods between 8 and 258 hours.
From these smoothed trajectories, a Hilbert transform was used estimate points at which the phase crossed $0$ to find period and amplitude coefficients of variation.
An additional noise parameter, the standard deviation in the $(1 \text{hr}, 8 \text{hr})$ wavelet components divided by the overall rhythm amplitude, was used to quantify the high-frequency noise of the system.
This wavelet component was chosen as it contained only the highest-frequency noise in the signal's spectrum.
From these three noise variables, a combined noise metric was constructed by projecting the variables along their first principle component (using scikit-learn, \url{http://scikit-learn.org/}).
Cells were ranked according to this combined noise metric, and a high-noise group and low-noise group were constructed by taking the 39 highest-noise and lowest-noise cells, respectively.
The raw bioluminescence profiles were not initially synchronized, so the traces were offset to have the same starting phase in order to simulate the gradual desynchronization of a group of oscillators. 
This was accomplished by starting each trace at the first phase zero-crossing, found using a Hilbert transform.

\subsubsection*{Bootstrap estimations of the damping rate difference}
Averaged traces for low and high-noise group displayed a damped sinusoidal rhythm.
The first 4 days of rhythms ($T_s = 0.5,\ N = 192$) were fit using a damped sinusoid.
To ensure the difference in damping rate between groups was statistically significant, a bootstrap analysis was performed.
In each of $10,000$ bootstrap trials, cells were randomly assigned evenly to either the low-noise or the high-noise group (one cell was randomly omitted in each trial to ensure even group sizes).
The absolute difference in damping rate between the two populations was recorded to yield a two-tailed test.
The observed test statistic, $|d_h - d_l| = 6.65\times10^{-3}$, was found to be significant at the $\alpha = 0.05$ confidence threshold ($p = 0.0264$).

\subsection*{Quantifying dose-dependent effects of small molecule modulators}
Bioluminescence traces ($T_s = 1.67,\ N=71$) with increasing small molecule concentrations were fit with a damped sinusoid using the method described in a previous section.
Because the small molecules were toxic at very high concentrations, experiments were removed from further analysis where $R^2 < 0.80$ (Figure~\ref{fig:small_molecule_r2}).

\subsection*{{\itshape In silico} prediction of small molecule experiments}
A previously published mathematical model of circadian rhythms \cite{Hirota2012} was used to predict the effects on population damping rate from the dose-dependent small molecule experiments.
The parameters used to capture the effects of each small molecule were the same as described previously \cite{St.John2014}.
The model was converted to a stochastic biochemical system and subsequently simulated using StochKit2 \cite{Sanft2011a} (via GillesPy, \url{https://github.com/JohnAbel/gillespy}).
Population-level rhythms were found by taking the average of $1,000$ noninteracting oscillators, starting at identical initial conditions.
The only parameter left unspecified by the deterministic model was the cell volume, $\Omega$, which controlled the amount of noise in the system.
For each $\Omega$, a damped sinusoid was fit to the population-averaged state trajectory. 
An $R^2$ value was calculated for each fit, taking into account all eight state variables.

\subsubsection*{Fitting the volume parameter}
We calculated an average experimental damping rate of $d = 0.0151$ from the $0 \mu M$ bioluminescence trajectories for both KL001 and longdaysin.
{\itshape In silico} damping rates were calculated for logarithmically spaced values of $\Omega \in (100, 500)$.
Ten independent groups of $1,000$ oscillators were simulated for each $\Omega$, from which the means and standard errors were found.
Simulations in which $R^2 < 0.90$ were removed from further analysis.
A weighted least-squares regression (using statsmodels, \url{http://statsmodels.sourceforge.net/}) was performed for $\log d$ vs.\ $\log \Omega$, using the $\log \text{SEM}$ of each measurement as a regression weight (Figure~\ref{fig:vol_calibration}).
The best fit was found to be $\Omega = 226.3 \pm 9.0$, and was used for subsequent model simulations.

\subsubsection*{Parameter knockdown experiments}
We replicated the effects of the small molecules KL001 and longdaysin mathematically through the reductions of the $vdcn$ and $vac1p$ parameters, respectively (Tables~\ref{tab:model}-\ref{tab:parset}).
Knockdown simulations were performed with $20$ values of each parameter, linearly spaced between $100\%$ and $15\%$ of their nominal value.
Similar to the volume calibration simulations, $10$ independent populations of $1,000$ oscillators were simulated from an initially entrained state.
Means and standard errors in period and damping rate were calculated from each population-averaged trajectory.
Simulations in which $R^2 < 0.90$ were removed from further analysis.

\subsection*{Fitting the genome-wide siRNA screen}
We analyzed the data and annotations for the $111,743$ wells ($T_s = 2,\ N = 72$) in the Zhang {\itshape et al.,} 2009 screen \cite{Zhang2009}.
Fits for which the $R^2 < 0.80$ were discarded.
The natural logarithm of the amplitude was used, since it more closely resembled a normal distribution and was on a similar scale to the damping rate.
Plate to plate variation, as shown in Figure~\ref{fig:plate_variation}, was more severe than variation on a well-to-well basis, Figure~\ref{fig:well_variation}.
Parameters were therefore normalized on a plate-by-plate basis using a robust z-score \cite{Birmingham2009}:
\[
  z_{R,i} = \frac{p_i - M(p_i)}{M\left(\left|p_i - M(p_i)\right|\right)}, \qquad i \in \{0, \dots, {\cal P} - 1 \}
\]
where $M(\cdot)$ denotes the median of a vector, and $p_i$ contains all the points in one plate, and $\cal P$ is the number of plates.
We removed outlier points prior to calculating the moments of the distributions, Pearson correlation coefficients, and performing the multivariable linear regression.
Outliers were defined as points which contained a z-metric (in either period, phase, amplitude, or damping rate) with an absolute value greater than eight.
We chose the ``control'' wells to be those which contained no siRNA, as these proved to be more numerous than those containing reference siRNA perturbations and were clustered similarly to the highest-density regions of the perturbed fits.

\subsubsection*{Detecting outlier perturbations}
We found the average response to each gene perturbation by grouping the perturbed dataset by target gene ID (using pandas, \url{http://pandas.pydata.org/}).
A Hotelling's $T^2$ test was used to determine whether the means of each gene knockdown was significantly different from the control population.
A robust covariance estimator was used to find the location and covariance of the control distribution (using scikit-learn).
Because the control distribution ($n=11,253$) is much larger than that for any particular gene ID ($n \approx 4$), the effect of the perturbed sample on the pooled covariance was neglected.

\subsection*{Software}
All computations were performed using Python.
Code used to perform the analysis and produce the figures in this manuscript can be found online at \url{https://github.com/pstjohn/decay-methods}.



\section*{Conclusion}

% Our method assumes independent and sustained oscillators at the single-cell level.
% While these assumptions have not always been upheld in the circadian literature \cite{Izumo2003}, more recent studies have demonstrated the validity of these assumptions in cultured fibroblast cells.
% Earlier mathematical studies have analysed on the damping rate of oscillators without assuming independent or homogeneous populations \cite{Rougemont2007}.
% However, such approaches introduce many additional fitted parameters, limiting their applicability to high-throughput screens.


In this study, we have described a method by which changes to stochastic noise can be estimated from population-level circadian bioluminescence recordings.
An overview of the computational steps involved in this method are outlined in Figure~\ref{fig:comp_overview}.
While the method can be applied to existing experimental data, there are also practical considerations for the design of future experiments.
Because the damping rate must be inferred from the time-varying amplitude, collecting bioluminescence data for longer time periods yields more accurate results and reduces the potential impact of initial transient regions.
Additionally, a sampling rate that is high enough to confidently capture the peaks and troughs of gene expression is required -- in this study, the 2 hr sampling window of the siRNA screen proved sufficient.
While achieving such a rate is typically not difficult for bioluminescence experiments, it may limit the method's applicability in experiments where samples need to be analyzed at each time point.
We also note that there are many available tools for detrending and regressing time-series data.
While we prioritized computationally efficient methods (which could be scaled to genome-wide screens), the best methods for any particular application may vary depending on the data.

As high amplitude circadian oscillations are important for maintaining metabolic health, many studies have sought to find small molecule candidates which increase oscillatory amplitude \cite{Chen2013}.
In the search for circadian clock therapeutics, high-throughput methods are frequently used to screen for such drugs, often neglecting potential effects on cell-autonomous noise.
While we have demonstrated a method by which the effects of small molecules on noise can be inferred from high-throughput methods, we have also shown that the potential improvement of clock robustness may be limited.
Amplitudes of circadian rhythms may therefore be best increased by small molecule therapies which act transiently to synchronize peripheral oscillators.
While such a method would require accurate alignment of drug administration to the correct circadian phase, a recent {\itshape in silico} study has demonstrated the potential effectiveness of such an approach in improving amplitudes in peripheral tissues \cite{St.John2014a}.
Finally, the ability to extract an additional biologically relevant parameter from existing datasets will likely prove useful for many studies, as it allows further differentiation between perturbations which might otherwise have identical effects on bioluminescence rhythms.



\section*{Acknowledgments}
We thank Tsuyoshi Hirota, David Welsh, and John Abel for the helpful discussions, as well as the each of the authors of the original experiments for making their raw data available online.
This work was supported by the National Institutes of Health/National Institute of General Medical Sciences under award number 1R01GM096873-01 and by the Institute for Collaborative Biotechnologies through grant W911NF-09-0001 from the U.S.\ Army Research Office.


\bibliographystyle{plos2015}
\bibliography{condensed_library.bib}

\beginsupplement

\begin{figure}[tbp]
  \begin{center}
    % \includegraphics[width=0.5\textwidth]{figures/pdfs/FigS1.pdf}
  \end{center}
  \caption{{\bfseries Fit quality for the dose-dependent small molecule screens.} The bioluminescence rhythms in both reporter systems were well-described by a damped sinusoid. As the molecules were toxic to the reporter cells at high concentrations, fit quality declined with increasing dosage. Only fits with $R^2 > 0.8$ were kept for further analysis.}
\label{fig:small_molecule_r2}
\end{figure}

\begin{figure}[tbp]
  \begin{center}
    % \includegraphics[width=0.5\textwidth]{figures/pdfs/FigS2.pdf}
  \end{center}
  \caption{{\bfseries Calibration curve for fitting the volume parameter to experimental data.} The model's volume parameter was linearly related to the population-averaged damping rate on a log-log scale. Error bars represent the standard error of the mean, calculated by $10$ independent replicates for each volume. Points shown in gray had an average $R^2 < 0.9$ and were excluded from the linear regression. Solid and dashed grey lines indicate the mean and $95\%$ confidence intervals of the linear regression, respectively.}
\label{fig:vol_calibration}
\end{figure}

\begin{figure}[tbp]
  \begin{center}
    % \includegraphics[width=0.8\textwidth]{figures/pdfs/FigS3.pdf}
  \end{center}
  \caption{{\bfseries Plate-to-plate variation of fitted parameters in the Zhang {\itshape et al.,} 2009 genome-wide siRNA screen.} Dots indicate the median of each plate, with lines extending from the $5^\text{th}$ to $95^\text{th}$ percentile. While parameter fits were of similar magnitude for all plates, some inconsistencies were present. In order to accurately compare perturbations and controls between plate experiments, we normalized fitted parameters on a plate-by-plate basis.}
\label{fig:plate_variation}
\end{figure}

\begin{figure}[tbp]
  \begin{center}
    % \includegraphics[width=0.8\textwidth]{figures/pdfs/FigS4.pdf}
  \end{center}
  \caption{{\bfseries Well-to-well variation in the Zhang {\itshape et al.,} 2009 genome-wide siRNA screen.} Similar to Figure~\ref{fig:plate_variation}, dots indicate the median value of each parameter in each well, with lines extending from the $5^\text{th}$ to $95^\text{th}$ percentile. Well position did not seem to affect the fitted values, particularly in the middle regions which contained the siRNA knockdown library. Wells on either end showed significant variation, but these are likely due to the spotting of long- and short-period controls in the same position on each plate.}
\label{fig:well_variation}
\end{figure}

\begin{figure}[tbp]
  \begin{center}
    % \includegraphics[width=0.7\textwidth]{figures/pdfs/FigS5.pdf}
  \end{center}
  \caption{{\bfseries Variability in period length from cycle to cycle is greater than from cell to cell.} Using single-cell fibroblast data from \cite{Leise2012}, a distribution of period lengths were calculated for each cell. Average period length for each cell is shown by a blue dot, with cells sorted from lowest to highest average period. Error bars extend from the $5^\text{th}$ to $95^\text{th}$ percentile period within each cell. The gray shaded region extends from the $5^\text{th}$ to $95^\text{th}$ percentile of the average period lengths for the entire population. $87\%$ of cells show greater variability from cycle-to-cycle than the overall period heterogeneity.}
  \label{fig:cell2cell-variability}
\end{figure}

\begin{figure}[tbp]
  \begin{center}
    % \includegraphics[width=0.7\textwidth]{figures/pdfs/FigS6.pdf}
  \end{center}
  \caption{{\bfseries Period heterogeneity and phase diffusion display different damping profiles.} ({\itshape Left}) A simulated population of homogeneous, noisy oscillators dephases due to cycle-to-cycle variability. The envelope of the averaged expression is proportional to $t$.
  ({\itshape Right}) A deterministic, heterogeneous population dephases due to different free-running periods. In this case, the population displays ballistic phase diffusion, in which envelope of the population-averaged expression changes proportionally to $t^2$. (See Rougemont \& Naef, 2007 \cite{Rougemont2007} for a more detailed discussion).}
  \label{fig:dephasing-rates}
\end{figure}

\begin{figure}[tbp]
  \begin{center}
    % \includegraphics[width=0.7\textwidth]{figures/pdfs/FigS7.pdf}
  \end{center}
  \caption{{\bfseries Identifiability of intrinsic and extrinsic noise factors in population-level data.} A simple model with both intrinsic and extrinsic noise (Table~\ref{tab:novak}) was used to determine the identifiability of noise sources. Bioluminescence profiles were generated from a population of 1000 individual oscillators. The stochastic volume parameter, $\Omega$, controls the intrinsic noise, with larger volumes resulting in more deterministic profiles. Extrinsic noise was generated by using a normal distribution of free running periods, ${\cal N}(1, \sigma^2_T)$ days. Points represent window functions (amplitude over time) from 10 independent replications. Lines represent linear regressions, except in the right panel of part A, where a quadratic regression was used.
  ({\bfseries A}) For models with pure intrinsic (left) or pure extrinsic (right) noise sources, amplitude damping proportional to $e^{dt}$ vs.\ $e^{dt^2}$, respectively, was readily distinguishable. 
  ({\bfseries B}) For models with both intrinsic and extrinsic noise sources, it was not possible to differentiate between increasing intrinsic noise (left) and increasing extrinsic noise (right), as both showed nearly linear relationships between time and log amplitude.
  The reduction in consistency between points seen for very small amplitude oscillations represents a fundamental limitation of the method, as noise dominates when oscillations approach steady state.
}
  \label{fig:computational_dephasing}
\end{figure}

\begin{figure}[tbp]
  \begin{center}
    % \includegraphics[width=\textwidth]{figures/pdfs/FigS8.pdf}
  \end{center}
  \caption{{\bfseries Overview of the computational method.} ({\bfseries A}) Raw bioluminescence traces are collected from cultured cells. Since the method is mainly dependent on amplitude over time, longer experiments yield more identifiable damping rates. ({\bfseries B}) Raw traces are detrended and denoised. This can be accomplished using a variety of methods, including the discrete wavelet transform. ({\bfseries C}) The window of the oscillations (amplitude vs.\ time) is plotted on a semi-log axis to make the exponential damping apparent. Window functions can be calculated using a Hilbert transform, or simply by using the absolute value of the local peaks (as was done here). A linear regression of the window function in semi-log space will yield both the initial amplitude (intercept) and damping rate (slope), with care being taken to avoid the over-influence of outlier points. Standard statistical techniques could then be used to verify that differences in slope are indeed significant.}
  \label{fig:comp_overview}
\end{figure}



\begin{table}[p]
  \centering
  \caption{{\bfseries A model of the mammalian core circadian feedback loop, from Hirota {\itshape et al.,} 2012 \cite{Hirota2012}.} Lower case letters (p: {\it Per}, c1: {\it Cry1}, c2: {\it Cry2}) are mRNA state variables. Uppercase letters (P: PER, C1: CRY1, C2: CRY2) are the free (cytosolic) proteins. C1N: CRY1 and C2N: CRY2 are the nuclear proteins.}

  \begin{align}
    \label{eq:p}
    \frac{d\mathrm{\bf p}}{dt}    & = \frac{v_{\text{txn,{\bf p}}}}{k_{\text{txn,{\bf p}}} + \left(\text{\bf C1N} + \text{\bf C2N} \right)^3} - \frac{v_{\text{deg,{\bf p}}} \;\text{\bf p} }{k_{\text{deg,{\bf p}}} +\text{\bf p} } \\
        %
    \label{eq:c1}
    \frac{d\text{\bf c1}}{dt}   & = \frac{v_{\text{txn,{\bf c1}}}}{k_{\text{txn,{\bf c}}} + \left(\text{\bf C1N} + \text{\bf C2N} \right)^3} - \frac{v_{\text{deg,{\bf c1}}} \;\text{\bf c1} }{k_{\text{deg,{\bf c}}} +\text{\bf c1} } \\
        %
    \label{eq:c2}
    \frac{d\text{\bf c2}}{dt}   & = \frac{v_{\text{txn,{\bf c2}}}}{k_{\text{txn,{\bf c}}} + \left(\text{\bf C1N} + \text{\bf C2N} \right)^3} - \frac{v_{\text{deg,{\bf c2}}} \;\text{\bf c2} }{k_{\text{deg,{\bf c}}} +\text{\bf c2} } \\
        %
    \begin{split}
      \frac{d\text{\bf P}}{dt}  & = k_\text{tln,{\bf p}} \;\text{\bf p}  - \frac{v_{\text{deg,{\bf P}}} \;\text{\bf P} }{k_{\text{deg,{\bf P}}} +\text{\bf P} } - v_{\text{a,{\bf CP}}} \;\text{\bf P}  \;\text{\bf C1}  + v_{\text{d,{\bf CP}}} \;\text{\bf C1N} \\
      & \quad - v_{\text{a,{\bf CP}}} \;\text{\bf P}  \;\text{\bf C2}  + v_{\text{d,{\bf CP}}} \; \text{\bf C2N}
    \end{split} \\
        %
    \frac{d\text{\bf C1}}{dt}   & =\text{\bf c1}  - \frac{v_{\text{deg,{\bf C1}}} \;\text{\bf C1} }{k_{\text{deg,{\bf C}}} +\text{\bf C1} } - v_{\text{a,{\bf CP}}} \;\text{\bf P}  \;\text{\bf C1}  + v_{\text{d,{\bf CP}}} \;\text{\bf C1N} \\
        %
    \frac{d\text{\bf C2}}{dt}   & =\text{\bf c2} - \frac{ v_{\text{deg,{\bf C2}}} \;\text{\bf C2} }{k_{\text{deg,{\bf C}}} +\text{\bf C2} } - v_{\text{a,{\bf CP}}} \;\text{\bf P}  \;\text{\bf C2}  + v_{\text{d,{\bf CP}}} \; \text{\bf C2N} \\
        %
    \frac{d\text{\bf C1N}}{dt}  & = - \frac{ v_{\text{deg,{\bf CP}}} \;\text{\bf C1N} }{k_{\text{deg,{\bf CP}}} +\text{\bf C1N}  + \text{\bf C2N} } + v_{\text{a,{\bf CP}}} \;\text{\bf P}  \;\text{\bf C1} - v_{\text{d,{\bf CP}}} \;\text{\bf C1N} \\
        %
    \label{eq:C2N}
    \frac{d\text{\bf C2N }}{dt} & = - \frac{ \left(v_{\text{deg,{\bf CP}}} \; m_{\text{\bf C2N}}\right) \; \text{\bf C2N} }{k_{\text{deg,{\bf CP}}} + \text{\bf C2N}  +\text{\bf C1N} } + v_{\text{a,{\bf CP}}} \;\text{\bf P}  \;\text{\bf C2} - v_{\text{d,{\bf CP}}} \; \text{\bf C2N}
  \end{align}
  \label{tab:model}
\end{table}


\begin{table}[p]
  \caption{{\bfseries Parameter values for the model in Table~\ref{tab:model}.} Nominal values for the kinetic parameters are shown below, as published in Hirota {\itshape et al.,} 2012 \cite{Hirota2012}.}
  \label{tab:parset}
  \vspace{2mm}
  \centering
  \begin{tabular}{cllr} \toprule
       & Parameter                 & Description                       & Value \\ \midrule
    1  & $v_{\text{txn,{\bf p}}}$  & {\it Per} Transcription rate      & 0.195 \\
    2  & $v_{\text{txn,{\bf c1}}}$ & {\it Cry1} Transcription rate     & 0.131 \\
    3  & $v_{\text{txn,{\bf c2}}}$ & {\it Cry1} Transcription rate     & 0.114 \\
    4  & $k_{\text{txn,{\bf p}}}$  & {\it Per} Repression constant     & 0.425 \\
    5  & $k_{\text{txn,{\bf c}}}$  & {\it Cry1/2} Repression constant  & 0.259 \\
    6  & $v_{\text{deg,{\bf p}}}$  & {\it Per} Max degradation rate    & 0.326 \\
    7  & $v_{\text{deg,{\bf c1}}}$ & {\it Cry1} Max degradation rate   & 0.676 \\
    8  & $v_{\text{deg,{\bf c2}}}$ & {\it Cry2} Max degradation rate   & 0.608 \\
    9  & $k_{\text{deg,{\bf p}}}$  & {\it Per} Degradation constant    & 0.011 \\
    10 & $k_{\text{deg,{\bf c}}}$  & {\it Cry1/2} Degradation constant & 1.149 \\
    11 & $v_{\text{deg,{\bf P}}}$  & Max PERc degradation rate         & 2.970 \\
    12 & $k_{\text{deg,{\bf P}}}$  & PERc degradation constant         & 0.034 \\
    13 & $v_{\text{deg,{\bf C1}}}$ & Max CRY1c degradation rate        & 1.523 \\
    14 & $v_{\text{deg,{\bf C2}}}$ & Max CRY2c degradation rate        & 1.686 \\
    15 & $k_{\text{deg,{\bf C}}}$  & CRYc degradation constant         & 2.017 \\
    16 & $v_{\text{deg,{\bf CP}}}$ & CRYn degradation rate             & 0.101 \\
    17 & $m_{\text{\bf C2N}}$      & CRY2n degradation multiplier      & 3.318 \\
    18 & $k_{\text{deg,{\bf CP}}}$ & CRYn degradation constant         & 0.053 \\
    19 & $v_{\text{a,{\bf CP}}}$   & CRYn association rate             & 0.041 \\
    20 & $v_{\text{d,{\bf CP}}}$   & CRYn dissociation rate            & 0.002 \\
    21 & $k_{\text{tln,{\bf p}}}$  & PER translation rate              & 3.000 \\ \bottomrule
    \hline
  \end{tabular}
\end{table}

\begin{table}
  \caption{{\bfseries Simple model of transcription-translation oscillations with intrinsic and extrinsic noise.} A model with both intrinsic and extrinsic noise, developed to examine the effect of each noise source on population-level amplitudes. Intrinsic noise is generated by simulating the solution stochastically using GillesPy. Extrinsic noise is generated by varying the free-running period, $t_c$. Equations adapted from \cite{Novak2008}.}
  \centering
  \begin{align*}
    \frac{dX}{dt} &= t_c \left(\frac{1}{1 + Y} - X\right) \\
    \frac{dY}{dt} &= t_c \left(k_t \; X - k_d \; Y - \frac{Y}{\alpha_0 + \alpha_1 \; Y + \alpha_2 Y^2}\right)
  \end{align*}

  \begin{tabular}{rl}
    \toprule Parameter & Value \\\midrule
    $k_t$ & 20 \\
    $k_d$ & 1  \\
    $P$   & 4 (or 2)  \\
    $\alpha_0$ & 0.005 \\
    $\alpha_1$ & 0.05  \\
    $\alpha_2$ & 0.1   \\
    $t_c$ & 3.516 \\\bottomrule
    \end{tabular}
  \label{tab:novak}
\end{table}



\begin{table}
\begin{center}
\begin{tabular}{lclc}
\toprule
\textbf{Dep. Variable:}    &      Damping Rate       & \textbf{  R-squared:         } &$     0.169  $\\
\textbf{Model:}            &       OLS        & \textbf{  Adj. R-squared:    } &$     0.169  $\\
\textbf{Method:}           &  Least Squares   & \textbf{  F-statistic:       } &$     4782.  $\\
\textbf{Date:}             & Wed, 11 Feb 2015 & \textbf{  Prob (F-statistic):} &$     0.00   $\\
\textbf{Time:}             &     16:26:22     & \textbf{  Log-Likelihood:    } &$-1.7248e+05 $\\
\textbf{No. Observations:} &       94053      & \textbf{  AIC:               } &$ 3.450e+05  $\\
\textbf{Df Residuals:}     &       94048      & \textbf{  BIC:               } &$ 3.450e+05  $\\
\bottomrule
\end{tabular}\\[5ex]
\begin{tabular}{lccccc}\toprule
                   & \textbf{coef} & \textbf{std err} & \textbf{t} & \textbf{P$>$$|$t$|$} & \textbf{[95.0\% Conf. Int.]}  \\
\midrule
\textbf{intercept}     &     $-0.0370$ &       $0.014$    &   $-2.572$ &        $0.010$       &       $-0.065,  -0.009$      \\
\textbf{amplitude} &     $ 0.2375$ &       $0.003$    &   $86.282$ &        $0.000$       &       $ 0.232,   0.243$      \\
\textbf{period}    &     $-0.1521$ &       $0.003$    &   $56.798$ &        $0.000$       &       $-0.157,  -0.147$      \\
\textbf{phase}     &     $-0.2354$ &       $0.003$    &   $85.598$ &        $0.000$       &       $-0.241,  -0.230$      \\
\textbf{perturbation type}      &     $ 0.3197$ &       $0.015$    &   $20.664$ &        $0.000$       &       $ 0.289,   0.350$      \\
\bottomrule
\end{tabular}\\[5ex]
\begin{tabular}{lclc}\toprule
\textbf{Omnibus:}       &$9769.391$& \textbf{  Durbin-Watson:     } &$    1.876$ \\
\textbf{Prob(Omnibus):} &$  0.000 $& \textbf{  Jarque-Bera (JB):  } &$18459.719$ \\
\textbf{Skewness:}          &$  0.697 $& \textbf{  Prob(JB):          } &$     0.00$ \\
\textbf{Kurtosis:}      &$  4.664 $& \textbf{  Cond. No.          } &$     8.34$ \\
\bottomrule
\end{tabular}
\end{center} 
\caption{{\bfseries Multivariable linear regression results.} Fit statistics which demonstrate the effect of each other fitted parameter on damping rate. Of particular note is the perturbation type categorical variable, which demonstrates that the presence of siRNA perturbation increases damping rate on average, controlling for changes in other variables. Higher amplitude is also correlated with higher damping rate. However, in total damping rate is poorly predicted by the other fitted variables ($R^2 = 0.169$), indicating it describes an independent oscillatory feature.}
\label{tab:ols_reg}
\end{table}


\end{document}

